### DISCUSSION TITLE
Stripe Affirm blocked for payments below $50

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4154] [Q&A] [answered]
Hey guys,

Stripe disabled my affirm and marked as ineligble due to products being below $50. Is there a way to enable Affirm and Klarna (or any specific payment method) by minimum amount? I see the option "Minimum order total" for stripe payment method but its for anything being used for stripe. In other words, if I enable $50 and if the order is $40, it will say no payment methods available. Is there a way to fix this? If not, I think it would be great to be able to enable or disable certain payments by tickets with a minimum transaction threshold

**Comment by pc-coholic** [comment, marked as answer]
pretix already has code to not offer Affirm for orders under 50$ - so I'm a little bit confused here: your customers should never been have offered Affirm unless their order was higher than this threshold amount...

**Comment by jxs714** [comment]
Hi @pc-coholic - thanks for the update. I noticed that too after reaching out when I tested again. Not sure why, but I was blocked. Here was their message:

-------------------------------------
Affirm was enabled to make it easier for your customers to pay.	
 
Unfortunately, as a result of a routine review of your account, we’ve determined that your business falls into a category that is not supported by Affirm. This means that we will no longer be able to support Affirm transactions for STRIPE ACCOUNT NAME HERE. While you will no longer be able to process Affirm payments on your account, please note that neither your payouts nor your access to other payment methods are affected by this change. We’re very sorry we will no longer be able to support your use of Affirm. If you have any questions, please let us know and we’ll be happy to help.	
 
— The Stripe team
-------------------------------------

I spoke with them and they are looking into it. I received a transaction or two for affirm without any issue and they were above $100.

**Reply by pc-coholic** [reply]
Reading their message, I don't think the 50$ floor was an issue but rather what you were selling/how Stripe classified you as a merchant...

Let's see, what they'll offer as an explanation :)

**Reply by jxs714** [reply]
Hi @pc-coholic 

Thanks for the update. So their reasoning was because Stripe's specific terms and conditions (Their own conditions when using Klarna or Affirm) do not allow B2B. Apparently we cannot charge our vendors a vendor fee as they are a business. I guess they manually reviewed certain information and came to the conclusion to disable it. Thing is, Klarna and Affirm both allow B2B, so this issue is only because Stripe doesn't allow it through them. Hope this helps anyone who needs it :)
